cmd/version.go

```diff
@@ -81,3 +87,38 @@ func getPluginVersions(opts Options) []string {
 
 	return versions
 }
+
+// Checks GitHub releases and prints new version, if current version is outdated.
+// requires GitHub releases to follow semver.
+func (cli *CLI) printLatestReleaseVersion() {
+	latest, err := getLatestVersion()
+	if err != nil {
+		cli.formatter.Print(tflint.Issues{}, fmt.Errorf("Failed to check updates; %w", err), map[string][]byte{})
+	}
+	latestVersion, err := version.NewSemver(*latest.Name)
+	if err != nil {
+		cli.formatter.Print(tflint.Issues{}, fmt.Errorf("Failed to parse version; %w", err), map[string][]byte{})
+	}
+	compare := tflint.Version.Compare(latestVersion)
+	if compare < 0 {
+		fmt.Fprintf(cli.outStream, "New version available: %s\n", *latest.HTMLURL)
```

Comment:
Terraform prints, for example:

```
Your version of Terraform is out of date! The latest version
is 1.4.4. You can update by downloading from https://www.terraform.io/downloads.html
```

It would be nice to do something similar, namely:

* The new version
* The URL

You already have the latter here.